shoes-core/ext/install/Rakefile

```diff
@@ -18,12 +18,22 @@ require 'fileutils'
 task default: ['install_script']
 
 task :install_script do
+  # If we're run via jruby-complete.jar, bindir is within the jar itself so
+  # look for bin based on user directory. Otherwise, prefer Gem.bindir.
+  if Gem.bindir.start_with?("uri:classloader")
```

Comment:
I'd have preferred this to go to a separate query method but ntohing that'll halt me merging :)